backend/user_types.py

```diff
@@ -0,0 +1,13 @@
+from pydantic import BaseModel
+from typing import Optional, List
+
+
+class UserDetails(BaseModel):
+    email: str
+    password: str
+    selectedTherapist: Optional[str]
+
+class UserRegister(BaseModel):
+    user_type: str
```

Comment:
Maybe its better to do this field Boolean like: is-therapist: true

Reply:
I've updated to this ` user_type: Literal['therapist', 'patient']`. WDYT?